### DISCUSSION TITLE
How to hide the actions column for certain roles?

### DISCUSSION THREAD

**Discussion by SequelONE** [Laravel-Backpack/community-forum#705] [Q&A (Help)] [answered]
Hello,

Is it possible to completely hide the action column for users who have access to the admin panel? I am using the package PermissionManager.

**Second question**

Is it possible to move details_row_button.blade.php from left to right side in table?

**Comment by promatik** [comment, marked as answer]
Hi @SequelONE 🙌

> _Is it possible to completely hide the action column for users who have access to the admin panel?_

Yes!
Actually, we're about to merge a feature where you'll be able to grand or deny access to operations based on that entry, and the auth user.
You can read about it here; https://github.com/Laravel-Backpack/CRUD/pull/5335, it will be available very soon.

> _Is it possible to move details_row_button.blade.php from left to right side in table?_

Unfortunately, no (easily).
It may be possible, but it will not be easy at all. Datatables adds it there, and backpack hooks on it to adjust a lot of aspects in it, there's a bunch of CSS styles to align everything and deal with every case scenario (like having or not details row button and/or the dtr-control and/or the bulk selection checkboxes), you can mess with this, but get ready for many headaches 😬
